### ISSUE TITLE
[$25]No label associated with the 'search challenges' text box. The placeholder text disappears when user starts typing.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3242]
**Describe the bug**
No label associated with the 'search challenges' text box. The placeholder text disappears when user starts typing.

**To Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 
2. Start typing in the 'search challenges' text box. Notice that the placeholder disappears and there is no additional label which is present at all times.

**Expected behavior**

There should be a label visible at all times.

**Actual behavior**
Currently only a placeholder text has been given which disappears when user starts typing.

**Additional context**

Type: Challenge Listing Navigation

Page: https://www.topcoder.com/challenges

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099921 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099921 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
FYI: Please reference this guide:  https://www.w3.org/WAI/tutorials/forms/labels/

1. We need to have a form label for the challenge search box.
2. "Search Challenges" text should be visual, next to the Search bar (You can decide the best method to make it visually sound)

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099921 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099921 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by veshu** [comment]
Prize updated with respect to https://github.com/topcoder-platform/community-app/pull/3256#issuecomment-525136013

**Comment by tosha5252** [comment]
There is placeholder text before user starts to type...is it necessary that there still be placeholder text after user starts typing? @lakshmiathreya

**Comment by veshu** [comment]
> .is it necessary that there still be placeholder text after user starts typing?

@tosha5252 That is not possible to have placeholder even after user starts typing.

**Comment by lakshmiathreya** [comment]
@veshu pls clarify - since the specific request in this Issue is to make sure placeholder is seen after user starts typing - can we infer - this Issue cannot be fixed? Fyi - @tosha5252

**Comment by veshu** [comment]
@lakshmiathreya Yes we can't always show the placeholder. This is default behavior of input control. as solution for the expected result "There should be a label visible at all times."

There is label before the search box as suggested here https://github.com/topcoder-platform/community-app/issues/3242#issuecomment-523721688
 
![image](https://user-images.githubusercontent.com/2202045/64309882-c311c100-cfbd-11e9-87c5-d8b6626864b2.png)

**Comment by drasticdpk** [comment]
Pass as expected. FYI @lakshmiathreya @tosha5252

**Comment by lakshmiathreya** [comment]
Label for 'Search Challenges' before the search box is added. Verified on MacOS/Chome and Win10/Chrome. Placeholder text in text input box can't be visible after user starts typing - as clarified above.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099921<br/><br/>```This is an automated message for crazyk via Topcoder X```
